app/views/admin/cases/show.html.slim

```diff
@@ -97,8 +97,10 @@
       - if @case.in_progress?
         .text-dark.justify-content-between.mb-2.align-items-center
           strong Actions
-        = link_to "Resolve", admin_case_path(@case, status: Case::RESOLVED), method: :patch, class: 'btn btn-success mx-1 mb-2'
-        = link_to "Close", admin_case_path(@case, status: Case::CLOSED), method: :patch, class: 'btn btn-dark mx-1 mb-2'
+        = link_to "Approve", admin_case_path(@case, status: Case::RESOLVED), method: :patch, class: 'btn btn-success mx-1 mb-2'
```

Comment:
I originally named these `RESOLVED` and `CLOSED` to copy JIRA, in which the definitions are better defined here: https://community.atlassian.com/t5/Jira-questions/What-is-the-difference-between-closed-and-resolved-status-in/qaq-p/378505

I'm open to re-labeling, but I think @atsyed87 and his team should be aware.

Reply:
This was a request from Asad to assist with training 👍